helm_deploy/cla-public/templates/_helpers.tpl

```diff
@@ -61,3 +61,37 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
```

Comment:
Can we add a comment here on what this does like on line 54 - 56 e.g.

`{{/*
Create the name of the service account to use
*/}}`

Reply:
Good catch. Do you want to go see if cla_backend (where we shamelessly stole this from) has such a thing?
And then either suggest that here, or suggest some wording of your own

Reply:
I put a description up, can you check if its suitable @mayowaadeniyi

Reply:
Could we say `Takes environment variables set in the values and values-environment file, and arranges them in an appropriate format`?

Reply:
Did you leave `What does this pull request do` and `Any other changes that would benefit highlighting?` blank and put your message outside of those headers intentionally?

Also can you mention how to turn on emergency message? (is there a kubectl command?)

Reply:
I see what I did there @said-moj  and have corrected that, I have instructions ready to go into confluence for this, but also put the command to edit the config map into the description.